requirements.txt

```diff
@@ -1,4 +1,4 @@
-mongomock
+mongomock<=3.0.0
```

Comment:
I guess you really want a mongomock version less than and not less than equal 3.0 right?